lib/src/code_field/code_field.dart

```diff
@@ -75,6 +76,16 @@ final _shortcuts = {
     LogicalKeyboardKey.shift,
     LogicalKeyboardKey.tab,
   ): const OutdentIntent(),
+
+  // Comment Uncomment
+  LogicalKeySet(
+    LogicalKeyboardKey.control,
+    LogicalKeyboardKey.slash,
+  ): const CommentUncommentIntent(),
+  const SingleActivator(
+    LogicalKeyboardKey.slash,
+    meta: true,
+  ): const CommentUncommentIntent(),
```

Comment:
Due to this issue https://github.com/flutter/flutter/issues/103964
I suggest to also replace other `LogicalKeySet`-s  that have `meta` with `SingleActivator`